tests/langchain/test_langchain_model_export.py

```diff
@@ -650,29 +647,13 @@ def test_langchain_agent_model_predict_stream():
     assert inspect.isgenerator(response)
     assert list(response) == [
         {
-            "actions": [
```

Comment:
note: The spec of agent response model changes frequently so making the assertion less strict.